CONTRIBUTING.md

```diff
@@ -78,6 +78,25 @@ If you plan to submit a pull request for anything more than a typo or obvious bu
 Depending on the size of the feature you may be expected to first write a design proposal.
 A proposal template is [available here](https://github.com/heptio/contour/tree/master/design/design-document-tmpl.md)
 
+### Rules of thumb for commits
```

Comment:
s/ Rules of thumb for commits/Commit Message Guidelines

I'd like to avoid _rule of thumb_ as (as I found out recently) in some jurisdictions it doesn't mean what I think you meant. https://en.wikipedia.org/wiki/Rule_of_thumb. I know you didn't mean any harm by this, but we should probably avoid this term.

Reply:
whoa, okay, I had no idea about that history. Will avoid in the future, thanks.

Reply:
Yeah, it was a surprise to me as well. I figure it’s better to be safe than sorry.